test/e2e/specs/find-dojo.spec.js

```diff
@@ -63,7 +66,19 @@ describe('Find Dojo page', () => {
     browser.back();
 
     DojoPage.detectLocationButton.waitForVisible();
-    expect(DojoPage.dojoListItemNames.length).to.equal(5);
+    expect(DojoPage.dojoListItemNames.length).to.equal(6);
+  });
+
+  it('should paginate search results', () => {
+    DojoPage.openWithQuery('dublin');
+    DojoPage.showDojoListCount.waitForVisible();
+    expect(DojoPage.showDojoListCount.getText()).to.equal(`Showing 1 to 6 of ${db.dojos.length} Dojos`);
+    expect(DojoPage.dojoListItemNames.length).to.equal(6);
+
+    DojoPage.paginationButtonPageTwo.click();
+    DojoPage.showDojoListCount.waitForVisible();
+    expect(DojoPage.showDojoListCount.getText()).to.equal(`Showing 7 to 8 of ${db.dojos.length} Dojos`);
```

Comment:
I guess the `8` here should probably be `${Math.min(db.dojos.length, 12)}` in keeping with not needing to update tests when we add new mock data.

Reply:
Why the Math.min/12 ?

Reply:
Because right now, we have 8 dojos, so we show `7 to 8`. When we have 12, we'll show `7 to 12`. When we have 13, we'll still be showing `7 to 12` because the 13th will be on the next page, hence the desire for the `Math.min`.

Reply:
oh, this 8. Yes. sorry :)